tools/topology/CMakeLists.txt

```diff
@@ -132,6 +132,8 @@ set(TPLGS
 	"sof-cml-demux-rt5682-max98357a\;sof-cml-demux-rt5682-max98357a\;-DPLATFORM=cml"
 	"sof-cml-rt1011-rt5682\;sof-cml-rt1011-rt5682\;-DPLATFORM=cml\;-DPPROC=volume"
 	"sof-cml-rt1011-rt5682\;sof-cml-eq-rt1011-rt5682\;-DPLATFORM=cml\;-DPPROC=eq-iir-eq-fir-volume"
+	"sof-cml-rt1011-rt5682-nokwd\;sof-cml-rt1011-rt5682-nokwd\;-DCHANNELS=2\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4"
```

Comment:
Do we have machine driver use the tplg by its name `sof-cml-rt1011-rt5682-nokwd.tplg`

Reply:
No, this is not loaded by machine driver, the plain sof-cml-rt1011-rt5682.tplg is used so this needs to be copied over  it. This could go to development, I'll check how it works for me.

Reply:
@xiulipan I just updated the PR with this topology moved to development. There was no changes needed to topology so this works well.

Reply:
@singalsu Thanks for the update. I think we can do some clean up to move some experiment audio processing tplg to development folder.